socorro/cron/crontabber_app.py

```diff
@@ -3,67 +3,109 @@
-socorro.cron.jobs.clean_raw_adi.CleanRawADICronApp|1d
-socorro.cron.jobs.clean_missing_symbols.CleanMissingSymbolsCronApp|1d
-socorro.cron.jobs.missingsymbols.MissingSymbolsCronApp|1d
-socorro.cron.jobs.featured_versions_automatic.FeaturedVersionsAutomaticCronApp|1h
-socorro.cron.jobs.upload_crash_report_json_schema.UploadCrashReportJSONSchemaCronApp|1h
-'''
+DEFAULT_JOBS_BASE = [
+    'socorro.cron.jobs.weekly_reports_partitions.WeeklyReportsPartitionsCronApp|7d',
+    'socorro.cron.jobs.matviews.ProductVersionsCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.SignaturesCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.RawUpdateChannelCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.ADUCronApp|1d|08:30',
+    'socorro.cron.jobs.matviews.DuplicatesCronApp|1h',
+    'socorro.cron.jobs.matviews.ReportsCleanCronApp|1h',
+    'socorro.cron.jobs.bugzilla.BugzillaCronApp|1h',
+    'socorro.cron.jobs.matviews.BuildADUCronApp|1d|08:30',
+    'socorro.cron.jobs.matviews.AndroidDevicesCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.GraphicsDeviceCronApp|1d|05:00',
+    'socorro.cron.jobs.ftpscraper.FTPScraperCronApp|1h',
+    'socorro.cron.jobs.elasticsearch_cleanup.ElasticsearchCleanupCronApp|7d',
+    'socorro.cron.jobs.drop_old_partitions.DropOldPartitionsCronApp|7d',
+    'socorro.cron.jobs.truncate_partitions.TruncatePartitionsCronApp|7d',
+    'socorro.cron.jobs.clean_raw_adi_logs.CleanRawADILogsCronApp|1d',
+    'socorro.cron.jobs.clean_raw_adi.CleanRawADICronApp|1d',
+    'socorro.cron.jobs.clean_missing_symbols.CleanMissingSymbolsCronApp|1d',
+    'socorro.cron.jobs.missingsymbols.MissingSymbolsCronApp|1d',
+    'socorro.cron.jobs.featured_versions_automatic.FeaturedVersionsAutomaticCronApp|1h',
+    'socorro.cron.jobs.upload_crash_report_json_schema.UploadCrashReportJSONSchemaCronApp|1h',
+]
+
+DEFAULT_JOBS = ', '.join(DEFAULT_JOBS_BASE)
+STAGE_JOBS = ', '.join(
+    DEFAULT_JOBS_BASE +
+    ['socorro.cron.jobs.fetch_adi_from_hive.FAKEFetchADIFromHiveCronApp|1d']
+)
+
+
+def jobs_converter(path_or_jobs):
+    """Takes a Python dotted path or a jobs spec and returns crontabber jobs
+
+    Example Python dotted path::
+
+        jobs_converter('socorro.cron.crontabber_app.DEFAULT_JOBS')
+
+    Example jobs spec::
+
+        jobs_converter('socorro.cron.jobs.ftpscraper.FTPScraperCronApp|1h')
+
+
+    :arg str path_or_jobs: a Python dotted path or a crontabber jobs spec
+
+    :returns: crontabber jobs InnerClassList
+
+    """
+    if '|' not in path_or_jobs:
+        # NOTE(willkg): crontabber's class_converter returns the value pointed
+        # to by a Python dotted path
+        input_str = class_converter(path_or_jobs)
+    else:
+        input_str = path_or_jobs
+
+    from_string_converter = classes_in_namespaces_converter_with_compression(
+        reference_namespace=Namespace(),
+        list_splitter_fn=line_splitter,
+        class_extractor=pipe_splitter,
+        extra_extractor=get_extra_as_options
+    )
```

Comment:
This `from_string_converter` is what the `jobs` configuration variable already used:

https://github.com/mozilla/crontabber/blob/57c463a521cdd2f9db4ae3c1104559abf88d1607/crontabber/app.py#L679